week2/Week2/src/week2/EvenOrOdd.java

```diff
@@ -0,0 +1,32 @@
+
+package week2;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author sara
+ */
+public class EvenOrOdd {
+
+    public static void printNum(int number){
+        if(number%2==0){
+            System.out.println("The Number is Even");
+        }
+        else if(number%2!=0){
```

Comment:
In this case, you don't need to use an "if - else if" statement. You can simplify it by removing the 'else if' statement and replacing it with an 'else' statement:
ex :
if (.....) {

} else {

}